iam-persistence/src/main/resources/db/migration/mysql/V100__alter_saved_user_auth_info_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE saved_user_auth_info MODIFY info_val varchar(256) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
Has this query some consequences on already present records?

Reply:
AFAIK, there should be no problem because all characters from `utf8mb4_unicode_ci` are present in `latin1_swedish_ci`. There might be problems with conversion if there were charset columns defined as foreign keys, but this is not the case.

I tried this query on our development instance (_iam-dev_) having that table already populated, and it succeeded.